Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst

```diff
@@ -161,7 +161,7 @@ Both Data and Commands Executed With Alternative User Object
 
 In this case it is shown how you can use the same object instance to
 submit both data and execute commands if you like. The order will
-depend on the order of line 4 and 5.
+depend on the order of line 3 and 4.
```

Comment:
Same here: I'd mention the two methods directly instead.

Reply:
same here